lerna.json

```diff
@@ -5,7 +5,7 @@
   "version": "independent",
   "command": {
     "version": {
-      "allowBranch": ["master", "develop", "workflows/require-e2e-before-build-release"],
+      "allowBranch": ["master", "develop", "workflows/require-e2e-before-build-release", "chore/lerna-publishing-config"],
```

Comment:
Should we remove these two? `workflows/require-e2e-before-build-release` and `chore/lerna-publishing-config`

Reply:
I don't think so. Of course I'll remove the last one after I transform this draft to PR, but `workflows/require-e2e-before-build-release` I'm not sure whether we still use. @Kacper-RF can you confirm?

Reply:
I think we don't use it at the moment, we triggering E2E tests before releasing but from `build-release.yml` workflow as separate jobs.

Reply:
Ok, then I'll remove it from here